tests/conftest.py

```diff
@@ -24,32 +23,60 @@
 def TestModel():
     """Provides a test model"""
 
-    class TestModel(MarshModel):
-        class Table:
-            name = 'peanut-butter'
-            hash_key = 'foo'
-            range_key = 'bar'
-            read = 5
-            write = 5
+    if 'marshmallow' in (os.getenv('SERIALIZATION_PKG') or ''):
```

Comment:
See overall comment below, but I think we _should_ keep the TestModel definitions separate and specific to the underlying validation scheme - this gives users the full power of marshmallow or schematics, without us giving them a consistent subset of both.